src/main/java/org/fundacionjala/coding/yerel/KataBankOCR.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.coding.yerel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * this class convert a String pipes and underscores
+ * and the fourth line is blank. Each account number should have 9 digits, all of which should be in the range 0-9.
+ */
+public class KataBankOCR {
+    private static final int NUMBER_TWENTYSEVEN = 27;
+    private static final int NUMBER_ZERO = 0;
+    private static final int NUMBER_ELEVEN = 11;
+    private static final int NUMBER_THREE = 3;
+    private static final HashMap<String, String> MAP = new HashMap<>();
```

Comment:
This should be 
private static final Map<String, String> MAP = new HashMap<>();